tests/integration/test_log_rotation.py

```diff
@@ -24,9 +26,9 @@
 
 METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 
-MYSQL_APP_NAME = "mysql-k8s"
+MYSQL_APP_NAME = MYSQL_DEFAULT_APP_NAME
 MYSQL_ROUTER_APP_NAME = "mysql-router-k8s"
```

Comment:
why MYSQL_ROUTER_APP_NAME was not moved to helpers? It looks half-baked.

Reply:
it does indeed look half baked, consistency is important. moved to helpers in 6e00cbc389baf29f68c607c218033729e8c37c21